ml/src/main/scala/ai/h2o/sparkling/ml/backend/H2OModel.scala

```diff
@@ -107,9 +113,13 @@ private[sparkling] object H2OModel extends RestCommunication {
         val stringElements = v.asScala.flatMap(stringifyJSON)
         val arrayAsString = stringElements.mkString("[", ", ", "]")
         Some(arrayAsString)
-      case _ =>
-        // don't put more complex type to output yet
-        None
+      case _: JsonNull => None
+      case v: JsonObject =>
+        if (v.has("name")) {
```

Comment:
To ensure that also complex values ( such as frame names, key names etc are return to the list) Mainly needed to discover validation frame